src/main/resources/org/jenkinsci/plugins/github_branch_source/TopicsTrait/config.jelly

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+  <f:entry title="${%Topics}" field="topicList">
```

Comment:
this is the data mapping issue
```suggestion
  <f:entry title="${%Topics}" field="topics">
```

Reply:
This makes sense, but then, `topicList` is a `String` while `topics` is an `ArrayList`.
In the `TopicsTrait` constructor, I then parse the ` String topicList` into the `ArrayList<String> topics` :thinking:

Reply:
ah i see

Reply:
Fixed in commit 81bde9b :+1: